src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,57 @@
 package core.basesyntax;
 
+import java.io.BufferedWriter;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
 public class WorkWithFile {
+    static final String SEPARATOR = ",";
+    static final int DESCRIPTION_OF_THE_LIST = 0;
+    static final int VALUE_OF_THE_LIST = 1;
```

Comment:
value, description - do not invent new words. Always use terminology from the task
![image](https://github.com/mate-academy/jv-work-with-file/assets/20326858/744fe4c4-4851-45b6-a421-54d4065ddc3b)
and why 'OF_THE_LIST'?